okhttp-tests/src/test/java/okhttp3/internal/tls/ClientAuthTest.java

```diff
@@ -41,16 +37,11 @@
 import static okhttp3.TestUtil.defaultClient;
 import static okhttp3.internal.platform.PlatformTest.getPlatform;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
 public final class ClientAuthTest {
   @Rule public final MockWebServer server = new MockWebServer();
 
-  public enum ClientAuth {
-    NONE, WANTS, NEEDS
-  }
```

Comment:
It was before!

Reply:
Yeah, I’m still in my 65K method limit mindset, where a private API doesn’t need the extra stuff.